src/DotNetProjectFile.Analyzers/Rule.cs

```diff
@@ -496,6 +496,16 @@ public static class Rule
         tags: ["Maintainability"],
         category: Category.CPM);
 
+    public static DiagnosticDescriptor UseVersionOnlyWithoutCpm => New(
+        id: 0804,
+        title: "Use Version only with Central Package Management not enabled",
+        message: "Use VersionOverride instead of Version when CPM is enabled.",
```

Comment:
I'm not a big fan of the message here, since ideally you shouldn't need any version overrides. This makes it sound like you _need_ to use `VersionOverride` when using CPM.

Reply:
You're right. I've changed it.